api_tests/nodes/views/test_node_links_detail.py

```diff
@@ -58,7 +59,7 @@ def test_returns_private_node_pointer_detail_logged_out(self):
         assert_equal(res.status_code, 200)
         target_node = res.json['data']['embeds']['target_node']
         assert_in('errors', target_node)
-        assert_equal(target_node['errors'][0]['detail'], 'Authentication credentials were not provided.')
+        assert_equal(target_node['errors'][0]['detail'], 'You do not have permission to perform this action.')
```

Comment:
I'd like to make this a separate PR; It seems that this was always the case that the wrong response was being provided here. The string originates from a 403 response that should be a 401.